charts/argo-cd/templates/argocd-configs/argocd-gpg-keys-cm.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  {{- if .Values.configs.gpgKeysAnnotations }}
+  annotations:
+    {{- range $key, $value := .Values.configs.gpgKeysAnnotations }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
+  {{- end }}
```

Comment:
```suggestion
  {{- with .Values.configs.gpgKeysAnnotations }}
  annotations:
    {{- range $key, $value := . }}
      {{ $key | nindent 4 }}: {{ $value | quote }}
    {{- end }}
  {{- end }}
```

Reply:
I'd prefer to use the same code style in all manifests inside the `argocd-configs` directory. Otherwise we should change the existing ones also.